src/util_lib/db.rs

```diff
@@ -224,13 +230,48 @@ impl FromColumn<QualifiedContractIdentifier> for QualifiedContractIdentifier {
     }
 }
 
+/// Make public keys loadable from a sqlite database
+impl FromColumn<Secp256k1PublicKey> for Secp256k1PublicKey {
```

Comment:
Is there a reason this isn't in `secp256k1.rs` anymore?

Reply:
Yes, `FromColumn` and `FromRow` aren't defined in the `stacks-common` crate, which is where `secp256k1` lives.